microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -548,6 +548,7 @@ See also:
 <!-- from RelNotes 111: -->
 
 * [CoreWebView2PermissionKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionkind)
+   <!-- * [CoreWebView2PermissionKind.WindowManagement Enum Value](/dotnet/api/microsoft.web.webview2.core.corewebview2permissionkind) -->
```

Comment:
@novac42 @peiche-jessica - Should 112 Overview of APIs list the specific enum value CoreWebView2PermissionKind.WindowManagement?

111 Overview didn't list enum value CoreWebView2PermissionKind.MidiSystemExclusiveMessages

Reply:
Just the Enum itself should suffice. If we want to list out enum values we should list out all of them.

Reply:
All the enum values:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2permissionkind?view=webview2-dotnet-1.0.1722.45&preserve-view=true#fields
Autoplay
Camera
ClipboardRead
FileReadWrite
Geolocation
LocalFonts
Microphone
MidiSystemExclusiveMessages
MultipleAutomaticDownloads
Notifications
OtherSensors
UnknownPermission
WindowManagement

Reply:
Removing the commented item.  Listing all the above enum members would be as long as the entire tab/section.